poptorch_experimental_addons/__init__.py

```diff
@@ -35,4 +35,4 @@ def _load_native_library() -> None:
 
 _load_native_library()
 
-from . import collectives, sparse  # NOQA:F401,E402
+from . import collectives, misc, sparse  # NOQA:F401,E402
```

Comment:
Personally I would prefer to use `custom_grad` directly from `pea` (`pea.custom_grad`). It strikes me as a self-contained, top-level, core functionality of the library. 

Seems a shame to "relegate" it to `misc`